osquery/tables/system/windows/tests/registry_tests.cpp

```diff
@@ -50,23 +50,24 @@ TEST_F(RegistryTablesTest, test_explode_registry_path_normal) {
 }
 
 TEST_F(RegistryTablesTest, test_registry_or_clause) {
-  SQL results("SELECT * FROM registry WHERE key = \"" + kTestKey +
-              "\" OR key = \"" + kTestSpecificKey + "\"");
-  auto testKeyFound = false;
-  auto specificKeyFound = false;
-  EXPECT_TRUE(results.rows().size() > 0);
-  for (const auto& row : results.rows()) {
-    auto key = row.at("key");
-    if (boost::starts_with(key, kTestSpecificKey)) {
-      specificKeyFound = true;
-    } else if (boost::starts_with(key, kTestKey)) {
-      testKeyFound = true;
-    } else {
-      assert(false);
-    }
-  }
-  EXPECT_TRUE(testKeyFound);
-  EXPECT_TRUE(specificKeyFound);
+  SQL result1("SELECT * FROM registry WHERE key = \"" + kTestKey + "\"");
+  SQL result2("SELECT * FROM registry WHERE key = \"" + kTestSpecificKey +
```

Comment:
Just so this is on record somewhere, we had a discussion about the `kTestSpecificKey` being the control panel, I think this is totally valid. A system will almost assuredly have a control panel registry key, and if it doesn't some other shit might be pretty off >.>

If other folks have better examples of a permanent specific registry key I'd be interested, but this should work good for now.